scripts/build-image.sh

```diff
@@ -39,5 +39,5 @@ IMAGE_NAME=$(grep vm_name $TEMPLATE | awk '{print $2}' | sed -e 's/\"//g' | sed
 set -xe
 cd packer
 packer build -var "chef_version=$CHEF_VERSION" $(basename $TEMPLATE)
-qemu-img convert -o compat=0.10 -O qcow2 -c ${DIR_NAME}/${IMAGE_NAME}.qcow2 \
+qemu-img convert -o compat=0.10 -O qcow2 -c ${DIR_NAME}/${IMAGE_NAME} \
```

Comment:
I suspect this is an issue with the version of packer. We're using a horribly outdated version but I'd rather we wait to deal with the upgrade path later. How about you set this change based on what version of packer is being executed (Can be seen running ``packer version``)? If we're running 0.7.5 then we need the .qcow2 extension, if we're running anything else let's assume we don't need it for now.